ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/ProfilesTableToggleGroup.tsx

```diff
@@ -16,11 +17,13 @@ export type ProfilesTableToggleGroupProps = {
 function ProfilesTableToggleGroup({ activeToggle }: ProfilesTableToggleGroupProps) {
     const { navigateWithScanConfigQuery } = useScanConfigRouter();
     const { profileName } = useParams();
+    const location = useLocation();
 
     const handleToggleChange = (resultsView) => {
+        const searchParams = qs.parse(location.search, { ignoreQueryPrefix: true });
```

Comment:
Is there a way to take advantage of `searchFilter` and `getQueryObject`? I haven't tested this but I assume it's going to pass all parameters and I worry that could bite us at some point.

Reply:
Actually even better, there's a function in `searchUtils` that includes the search prefix `getUrlQueryStringForSearchFilter`

Reply:
What is the `customParams`?

Reply:
I like the idea of using `searchFilter`. That would make it easier. Let me try your suggestions and see how it goes 👍🏼